src/ecalc/libraries/libecalc/common/tests/input/test_file_io.py

```diff
@@ -109,6 +120,46 @@ def test_no_nans(self, facility_resource_missing_value_file):
             == "csv file contains invalid data at row 1, all headers must be associated with a valid column value"
         )
 
+    @pytest.mark.parametrize(
+        "csv_line, is_valid_characters",
+        [
+            ("aa :, bb", True),
+            ("aa ., bb", True),
+            ("aa +, bb", True),
+            ("aa 0, bb", True),
+            ("aa _, bb", True),
+            ("aa -, bb", True),
+            ("aa /, bb", True),
+            ("aa #, bb", True),
+            ("aa @, bb", False),
+            ("aa %, bb", False),
+            ("aa &, bb", False),
+            ("aa ?, bb", False),
+            ("aa ', bb", False),
+            ('aa ", bb', False),
+            ("aa )(, bb", False),
+            ("aa ][, bb", False),
+            ("aa }{, bb", False),
+            ("aa ><, bb", False),
+        ],
+    )  # This is not meant to be extensive, just to test that we have some validation
+    def test_valid_characters(self, get_tmp_path, csv_line: str, is_valid_characters: bool):
```

Comment:
```suggestion
    def test_valid_characters(self, tmp_path_fixture, csv_line: str, is_valid_characters: bool):
```

I just started to and would like fixtures to be named fixture to easily see that it is a fixture. My suggestion is `<name of func>_fixture`, which I just did to one fixture. Better suggestions? Or reason not to?